### DISCUSSION TITLE
Search by URL/query string, tag by tag name

### DISCUSSION THREAD

**Discussion by pettijohn** [Kovah/LinkAce#655] [Ideas & Feature Requests]
Hi, cool project. I'm evaluating self-hosted bookmark options. One thing that would increase the utility of this would be GETtable URLs for search. E.g. /search?tag=programming+go that would return the intersection of tags that have programming AND go. It would allow [custom a search shortcut](https://support.google.com/chrome/answer/95426?hl=en&co=GENIE.Platform%3DDesktop) in a browser for quick access. 

Related, the primary key of tags should be the tag itself so that instead of https://demo.linkace.org/tags/95 you could have https://demo.linkace.org/tags/programming Or if rewriting your schema isn't on the roadmap, a new page just for tag search by name could solve it.

**Comment by joel72265** [comment]
Here are my 2 cents 

Search as you type by **Tags**, URL **(domain name**) and URL **description**. I don't know how LinkAce is structured but even if creating a flat file with domain names and tags which is searchable would be very useful.
